examples/terraform/azure/main.tf

```diff
@@ -16,6 +16,7 @@ limitations under the License.
 
 provider "azurerm" {
   features {}
+  skip_provider_registration = true
```

Comment:
Because it was giving exact error indicated in this link
```
The errors returned from the Azure API when a Resource Provider is unregistered are unclear (example API version '2019-01-01' was not found for 'Microsoft.Foo') 
```